apps/omg_watcher/test/support/integration/test_helper.ex

```diff
@@ -41,6 +41,14 @@ defmodule OMG.Watcher.Integration.TestHelper do
     utxos
   end
 
+  def wait_until_block_getter_fetches_block_after_current_child_block(timeout) do
```

Comment:
Make this name less specific. In some uses of this function we wait for exitor.

Reply:
I changed methods to:
`wait_for_current_block_fetch` - this function is more general so I didn't change for more specific name
`wait_for_block_fetch`